pkg/cluster/describe.go

```diff
@@ -89,6 +89,16 @@ func PrintClusterDesctipion(connection *sdk.Connection, cluster *cmv1.Cluster) e
 		shard = shardPath.Body().HiveConfig().Server()
 	}
 
+	var channelGroup string
+	version, err := connection.ClustersMgmt().V1().
```

Comment:
while this is technically correct implementation, we should have the channel group as a cluster resource attribute and the information should be derived from there.

Reply:
Sure, I will close this PR and rewrite it once the channel group is added as an attribute to the Cluster resource.